server/eclipse-project/src/main/resources/us/freeandfair/corla/default.properties

```diff
@@ -49,3 +49,6 @@ hibernate.c3p0.idle_test_period = 0
 
 windows_upload_file_location=c:\\tmp\\rla\\upload\\
 unix_upload_file_location=/tmp/corla/upload/
+
+# new random selection method that avoids CVR data as much as possible
```

Comment:
What do you think about saying something more, perhaps like this: 

```
# To use the ballot manifest for random ballot selection, set true; to use CVRs for random ballot selection, set false. 
# When using ballot manifests, we join to the CVRs - after selection - to provide ballot type information.
```

🤔 ...or maybe this should be documented in more detail in the runbook, or other suitable place...

Reply:
idk, this is only going to be around for one sprint, maybe I should add that note here?

Reply:
let's say a hair more here then.